profile/profile.yaml

```diff
@@ -0,0 +1,166 @@
+
+# Group your skills by level.
+# Your skills can be anything software related (e.g. language, frameworks, methodologies, software, etc.).
+# DO NOT mention skills that you don't want to work on in the future (e.g. MS office)
+skills:
+  # At least 5 skills
+  proficient:
+    - C/C++
+    - Linux/Bash
+    - RDBMS/SQL
+    - Agile/Scrum
+    - Git
+    - HTML 
+    - CSS
+    - BootStrap
+    - JavaScript
+    - Slack 
+    - Notion
+    - VIM 
+  # At least 5 skills
+  competent:
+    - Java
+    - SQL Developer
+    - Node.js 
+    - Postgres
+    - Heroku
+    - IBM i-series/AS400 
+    - PuTTY/FileZilla/WinSCP
+  # At least 5 skills
+  familiar:
+    - React 
+    - Handlebars 
+    - Sequalize
+    - WordPress 
+    - PHP
+    - Python
+    - JUCE 
+    - Arduino 
+    - Control Language
+    - MongoDB
+    - SSH/SCP/SFTP
+    - Windows/Powershell 
+    - XAMPP 
+    - Riverbed 
+    - UML 
+    - Rational Rose 
+    - StarUML 
+    - Sketch 
+
+# List of Jarvis projects
+jarvis_projects:
+  - name: "Cluster Monitor"
+    git_url: "/linux_sql"
+    # Please read the project description guideline carefully. Make sure you follow the simplified SDLC pattern. (t.ly/NlXQ)
+    # Please include the following keywords: CentOS, Bash/Shell, docker, RDBMS/SQL/Postgresql, crontab, data analytics
+    description: This project uses Google Cloud Platform(Compute Engine) to initialize a Virtual Machine with CentOS. Within the Virtual Machine, an instance of Docker is provisioned. A Postgresql instance is then provisioned on Docker. Data is inserted via an automated Bash/Shell script and retrieved from the RDBMS with SQL; which details hardware specifications. 
+  - name: "Core Java Apps"
+    git_url: "/core_java"
+    description:
+      #Keywords: Java 8, REST API, OAuth, Design patterns (e.g. DAO, MVC), JUnit, Mockito, Docker
+      - "Twitter App: Curabitur laoreet tristique leo, eget suscipit nisi. Sed in sodales ex. Maecenas vitae tincidunt dui, et eleifend quam."
+      # Keywords: Java 8, JDBC, RDBMS, SQL, CRUD,DAO
+      - "JDBC App: Curabitur laoreet tristique leo, eget suscipit nisi. Sed in sodales ex. Maecenas vitae tincidunt dui, et eleifend quam."
+      # Keywords: Java 8, Lambda, Stream API, IntelliJ
+      - "Grep App: Curabitur laoreet tristique leo, eget suscipit nisi. Sed in sodales ex. Maecenas vitae tincidunt dui, et eleifend quam."
+
+# List of personal or academic projects
+# Feel free to add more projects
+# Please read the project description guideline carefully. Make sure you follow the simplified SDLC pattern. (t.ly/NlXQ)
+highlighted_projects:
+    #project name
+  - name: "3-Band Equalizer"
+    #URL is optional
+    git_url: "https://github.com/Plum-Tree-Garden/JUCE/tree/main/3BandEQ"
+    # The project description must start with an action verb (https://bit.ly/3guO98u)
+    description: Virtual Studio Technology - Low Cut, High Cut and Parametric Equalizer with Frequency and Gain control.
```

Comment:
please mention some keywords, such as programming language and well-known libraries